### PR TITLE
Change texts, make them more readable

### PR DESCRIPTION
**Pull request by nienkedekker**

### CODE REVIEW

File: views/home.html.twig [hunk 1]
```diff
@@ -17,22 +17,24 @@
...
 <div class="info-page">
 <a name="info-page"></a>
+
 <div class="info-page-item">
-<center><span class="flaticon-like"></span></center>
-<h1>Unreliable</h1>
-<p>Don't complain if this goes down.</p>
+<center><span class="flaticon-user-3"></span></center>
+<h1>Anonymous</h1>
+<p>To sign up, you'll only need an e-mail address, a username and a password. We offer anonymous payments.</p>
```
Comment (EaterOfCode) [inline views/home.html.twig]: I laugh, atm we don't record email addresses and no payments is indeed anonymous

File: views/home.html.twig [hunk 2]
```diff
@@ -17,22 +17,24 @@
...
 </div>
 
 <div class="info-page-item">
-<center><span class="flaticon-garbage"></span></center>
-<h1>Incompetent</h1>
-<p>I have no idea what I'm doing dog meme.jpg</p>
+<center><span class="flaticon-like-1"></span></center>
+<h1>Open Source</h1>
+<p>We believe in the power of open source. Zer.ooo offers OpenVPN and is available for free on GitHub under a creative commons license. </p>
```
Comment (EaterOfCode) [inline views/home.html.twig]: Zer.ooo is MIT licensed (I need to add the LICENSE files to the repos)

File: views/home.html.twig (hunk 1 above)
Comment (nienkedekker) [inline views/home.html.twig]: Technically correct is the best kind of correct

File: views/home.html.twig (hunk 2 above)
Comment (nienkedekker) [inline views/home.html.twig]: I'll change this.

**Comment by EaterOfCode** [comment]
What about the title ~ Zer.ooo - Security At Its Worst ~ and ~ WHAT EVEN IS INFOSEC? ~ we may need to change that too, I'll tonight build a page with more in-depth details about the security

**Pull request merged**